components/adapters/firebug.js

```diff
@@ -0,0 +1,11 @@
+
+const SYSTEM = require("firebug.sdk/lib/core/system").System;
+
+
+exports.for = function (API) {
+
+	API.console.log("isDeveloperBrowser: " + SYSTEM.isDeveloperBrowser());
```

Comment:
Require the mapped package in our adapter module that will provide a nice boundary between PINF-abstracted code and SDKs. This boundary will in future inject intelligence logging code into the firebug.sdk (unless it already ships with it) to allow for deep insight into APIs when developing the PINF-abstracted code and running API codepaths.